src/util/nice-time.test.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-env jest */
+
+import niceTime from 'src/util/nice-time'
+
+describe('niceTime tests', () => {
+    test('should return now for timeperiod of less than 90 seconds', () => {
+        const date = new Date()
+        expect(niceTime(date)).toBe('now')
+    })
+
+    test('should return minutes for timeperiod of less than 600 seconds', () => {
+        const date = new Date()
+        const now = new Date()
+        now.setMinutes(date.getMinutes() + 20)
+        expect(niceTime(date, {now})).toBe('20 minutes ago')
+    })
+
+    test('should return timeperiod stamp for timeperiod less than 24 hours', () => {
+        const date = new Date()
+        const now = new Date()
+        now.setHours(date.getHours() + 5)
+        expect(niceTime(date, {now})).toBe(`${('0' + date.getHours()).slice(-2)}:${('0' + date.getMinutes()).slice(-2)}`)
+    })
+
+    test('should return the timeperiod stamp and the day for timeperiod less than 24 hours but not on the same day', () => {
+        const date = new Date()
+        const now = new Date()
+        now.setHours(date.getHours() + 22)
+        expect(niceTime(date, {now})).toBe(`Yesterday ${('0' + date.getHours()).slice(-2)}:${('0' + date.getMinutes()).slice(-2)}`)
```

Comment:
I thought the test should test all the cases of niceTime module. This would provide as a useful use case as well as make the system very stable. If we go the other way and just check with the single input, we might not come to know if some other part of the program gets broken.